config.json

```diff
@@ -483,6 +483,18 @@
       "unlocked_by": "scrabble-score",
       "uuid": "4b3f7771-c642-4278-a3d9-2fb958f26361"
     },
+    {
+      "core": false,
+      "difficulty": 6,
+      "slug": "rail-fence-cipher",
+      "topics": [
+        "strings",
+        "loops",
+        "conditionals"
+      ],
+      "unlocked_by": "atbash-cipher",
```

Comment:
`atbash-cipher` isn't a core exercise so it can't be unlocked by that. `rotational-cipher` would probably be a good exercise for it to be unlocked by? :)